engines/batch-ensembler/README.md

```diff
@@ -56,7 +56,7 @@ spec:
         options:
           viewsEnabled: "true"
           materializationDataset: dataset
-    joinOn:
+    join_on:
```

Comment:
Why is that? There is no a consistent naming convention with YAML (so both camel-case and snake-case are fine), but I use camel-case here (for instance, there is also `bqConfig ` at line:38).
Kubernetes uses camel case as well.

Reply:
Oh okay I didn't realise this, will change this back to camel.